server/views/pages/tagsTopicsSeries.html

```diff
@@ -47,26 +36,6 @@ <h1 id="title" class="govuk-heading-xl">{{ title }}</h1>
     data: data.relatedContent,
     lazyLoadContent: true
   }) }}
-
-
-  {% if not data.excludeFeedback %}
-    <div class="govuk-form-group govuk-hub-article-feedback govuk-hub-on-black-feedback">
```

Comment:
`govuk-hub-article-feedback` handled the border on this page type. Requires a `<hr>` as applied in the `tagsCategories.html`